source/index.html

```diff
@@ -226,4 +226,20 @@ <h2 class="text-5xl" id="sponsors">Sponsors</h2>
         </div>
     </div>
 </section>
+    
+<section class="max-w-full my-auto sm:mt-24 md:my-8 text-center py-4">
+    <h2 class="text-5xl" id="supporters">Supporters</h2>
+    <div class="font-semibold text-xl my-4">We receive non-financial help from other organisations too</div>
+    <div class="mx-auto max-w-full px-4">
+        <div class="grid gap-4 justify-items-center">
+            <a class="block bg-gray-200 p-2 rounded-lg no-underline" target="_blank" rel="noopener noreferrer" href="https://mule.to/p399">
+                <div class="mb-6 p-2">
+<!--                    <img class="h-24 rounded w-full object-contain object-center" src="/assets/sponsors/stickermule_logo_transparent.png" alt="Sticker Mule">-->
+                    <img class="h-24 rounded w-full object-contain object-center" src="https://user-images.githubusercontent.com/1196825/191993307-a6f931be-c9a9-4161-ab65-e0cbea5f3621.png" alt="Sticker Mule">
```

Comment:
For some reason image didn't work from Sculpin, so I uploaded it to GitHub. for now.